### DISCUSSION TITLE
22000.527 desktop broken!

### DISCUSSION THREAD

**Discussion by zyh941** [valinet/ExplorerPatcher#869] [Q&A]
In the past, everything ran well. I just updated my laptop to the latest stable windows version 22000.527, but unfortunately, desktop keeps black any empty. Start menu, Ctrl+ win+delete, win+I ( system settings) and file explorer do not work! Only I can run are tasker manager, services, PowerShell, cmd... Please help me! This is my primary laptop. 🥺
![85](https://user-images.githubusercontent.com/95530307/154262153-b0580343-b921-413b-84c2-e4b95a3af66a.png)

**Comment by Flexman** [comment]
Had the same issue, just restart Windows Explorer or File | Run | explorer from the task manager menu.

**Reply by zyh941** [reply]
I have tried restart file explorer, not work.

**Comment by jack-kylin** [comment]
Run ep_setup.exe /uninstall

**Reply by jack-kylin** [reply]
Or download the newest ep_setup.exe

**Reply by zyh941** [reply]
Can you tell me where's the default location of ep_setup.exe? The Windows cannot find 'ep_setup.exe'...

**Reply by valinet** [reply]
`ep_setup.exe` is the installer of EP. You downloaded that from GitHub probably. A copy of it is in `%PROGRAMFILES%\ExplorerPatcher`.

**Reply by Andre-10** [reply]
Had similar experience with the latest Windows 11 update. Black screen after windows restart.

1st, ... wait a minute, maybe new OS symbols file have to be downloaded..
2nd - restart Windows and then open ExplorerPatcher settings and restart explorer from there. If not working, set Taskbar to Windows 11, and after screen comes back, turn it back to Windows 10 style

you can run EP from Task manager via File / run new task and type 

rundll32.exe C:\Windows\dxgi.dll,ZZGUI

**Reply by DustyMustard** [reply]
Yeah, not fun for Windows 11. I want to offer this for clients but if an update goes to a blackscreen and they have to seek support it is a no go.

**Reply by valinet** [reply]
It's a pretty silly idea to have mission critical equipment running code that hacks together some legacy mode buried in some system programs by a dude doing this in his free time whose major (and kind of only) source of information regarding how it all works was disassembling of the system's binaries. Not to mention you wouldn't have any real support for when things go bad - I am supportive and try to fix things, but suppose I don't have time to repair something at some point, then what, your clients are screwed for that entire time...? Blows my mind...

A wiser idea would be to keep your clients running Windows 10 is you want a fully supported Windows 10 interface. This program is prone to "breaking", as y'all like to call it, it's by design and it should be pretty obvious from the way this works, which is described on these forums in plenty of places. I totally don't understand the disappointment. A lot of you here seem to get that, but I feel like you forgot that Microsoft is the source of blame for all of this, they have screwed up the interface, not me. Try to get in contact with them and maybe they will change their mind.

**Reply by DustyMustard** [reply]
In a perfect world Microsoft didn't f*ck a taskbar that has been a standard for 1 to several decades because their UI team wanted to feel secure in their job. And no-one is overlooking that you are pulling some code hacking magic to undo that nonsense.

Doesn't mean that I can't dream that you will hire a crew, come out with a company on nasdaq, and keep this going until Microsoft buys your company and you retire in Canada.
Or it doesn't mean that I don't fully understand how delicate what you are doing is, and that it might end at some point because Microsoft removes what is needed for this to be pulled off. But it is a reality that it would be nice if this was possible to offer more widely because it is good work and is something at least a few feel they really need.

**Reply by valinet** [reply]
Yeah, would be kind of nice, I guess. 😆 The thing is, with the money I made off of this, I'd be able to hire a crew for a couple of days. Retirement would be a long road ahead. The truth is. not a lot of people really suffer after this change, and this is kind of confirmed in practice. Most of the people I know use a combined taskbar in Windows 10 (most do not even really bothered to hide the useless search bar), and the few that I know who have upgraded to 11 haven't even noticed a change, other than taskbar icons being now centered rather than left-aligned.

So yeah, EP is a niche of a niche of a niche of a tiny niche when looking at Windows from a global perspective.
